build.gradle

```diff
@@ -416,9 +408,14 @@ configure(project.fineractJavaProjects) {
 
     // Configuration for the spotless plugin
     // https://github.com/diffplug/spotless/tree/main/plugin-gradle
+    dependencies {
+        spotbugsPlugins 'jp.skypencil.findbugs.slf4j:bug-pattern:1.5.0@jar'
+        spotbugs 'com.github.spotbugs:spotbugs:4.1.4'
+    }
+
     spotless {
         java {
-            targetExclude '**/build/**', '**/bin/**', '**/out/**'
+            targetExclude '**/build/**', '**/bin/**', '**/out/**', '**/integration-tests/**'
```

Comment:
why exclude integration-tests from Spotless now? Or are we already not covering them? Then we should separately fix that, first?

Reply:
True... was a bit late and was not sure where all the warnings/errors were coming from. I'll revert.

Reply:
Thanks! I have checked and see that this was done, so Resolving this point.